### PR TITLE
[SQL] Remove Radiological Review from Menu

### PR DESCRIPTION
**Pull request by ZainVirani**
This patch:

1. Removes menu permissions from `LorisMenuPermissions` for Radiological Review
2. Removes the menu item itself from `LorisMenu`

### CODE REVIEW

**Comment by ridz1208** [comment]
@ZainVirani you are setting SQL_NOTES to 1 but never resetting it to original value

**Comment by ridz1208** [comment]
@sruthymathew123 you will want to create the opposite patch for your IBIS production when upgrading

File: SQL/Archive/18.0/2018-01-18-remove-radiological-review-menu.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+WARNINGS;
```
Comment (driusan) [inline SQL/Archive/18.0/2018-01-18-remove-radiological-review-menu.sql]: shouldn't this go to the 19.0-dev branch?

**Comment by ZainVirani** [comment]
@ridz1208 `I've seen this place before my friend`
https://github.com/aces/Loris/pull/2798#pullrequestreview-37696229

File: SQL/Archive/18.0/2018-01-18-remove-radiological-review-menu.sql (hunk 1 above)
Comment (ZainVirani) [reply SQL/Archive/18.0/2018-01-18-remove-radiological-review-menu.sql]: if so, i'll have to close this PR and make another. lmk

**Comment by ridz1208** [comment]
@ZainVirani glad to know im consistent !!!

**Pull request closed without merging**